src/js/views/Login.jsx

```diff
@@ -21,6 +26,25 @@ function Login({ onLoginCallback, useLDAP }) {
 
   setDocumentTitle(t('login.signIn'))
 
+  function redirectToGoogle(e) {
+    e.preventDefault()
+    const redirectUri = new URL('/google/auth', globalState.baseURL).toString()
+    const scopes = [
+      'openid',
+      'https://www.googleapis.com/auth/userinfo.email',
+      'https://www.googleapis.com/auth/userinfo.profile'
+    ].join(' ')
+    document.location =
+      `${googleAuthorizationEndpoint}` +
+      `?client_id=${googleClientId}` +
+      `&redirect_uri=${redirectUri}` +
+      '&response_type=code' +
+      `&scope=${scopes}` +
+      '&access_type=offline' +
+      '&include_granted_scopes=true' +
+      '&prompt=select_account'
```

Comment:
My gut is telling me that the string should be a fully encoded URL though browser implementations tend to be pretty flexible.  The [spec for document.location](https://url.spec.whatwg.org/#concept-basic-url-parser) is pretty vague though the [path state](https://url.spec.whatwg.org/#path-state) and [query state](https://url.spec.whatwg.org/#query-state) both call out handling for percent encoded content:

![image](https://user-images.githubusercontent.com/350812/213026846-a9f154f3-abe0-4805-af4f-8435fe7af0a4.png)

The thing that made me pause and think about this is the inclusion of spaces in the scope parameter.  The valid [URL code points](https://url.spec.whatwg.org/#url-code-points) exclude the ASCII space (U+0020).  It might be valid to use a `+` instead of a space here but we might as well let a [`URLSearchParams` object](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) do the dirty work for us.

Maybe something like:
```
const args = new URLSearchParams()
args.append('client_id', googleClientId)
args.append('redirect_uri', redirectUri)
args.append('response_type', 'code')
args.append('scopes', scopes)
// ...
redirectUri.search = args.toString() // note that you assign to `search` and not `searchParams`
```